doc/api/next_api_changes/deprecations/25027-GL.rst

```diff
@@ -0,0 +1,4 @@
+``PolyQuadMesh.set_array()`` with compressed values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+... is deprecated.  Instead, pass the full 2D array of values
```

Comment:
This is note isn't useful to users as PolyQuadMesh is new.  I think you need something like the object returned from pcolor wants 2D values passed to set_array.  "Compressed values" isn't right either.  I think you need to be more loquacious here.  Maybe:

```
Object returned by `pcolor` has changed to `PolyQuadMesh`
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The old object was a `PolyCollection` with flattened vertices and array data.   
The new  `set_array` method on `PolyQuadMesh` requires a 2D array rather than a flattened 
array.  The old flattened data is still accepted, but deprecated.

```